includes/discovery/ipv6-addresses/eltex-mes21xx.inc.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * mes21xx.inc.php
+ *
+ * IPv6 address discovery file for eltex-mes21xx OS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  2021 Peca Nesovanovic
+ * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
+ */
+
+//standard MIB
+$oids = SnmpQuery::walk('IP-MIB::ipAddressIfIndex.ipv6')->table(2);
+//Radlan MIB
+$oidr = SnmpQuery::walk('RADLAN-IPv6::rlIpAddressTable')->table(2);
```

Comment:
You can merge the two arrays by adding the $oids variable as the second argument of ->table().  Will make things easier below.

Reply:
done